website/content/en/docs/building-operators/golang/testing.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Testing your Operator project
+linkTitle: Testing with EnvTest
+description: Learn how to ensure the quality of your Operator project 
+weight: 70
+---
+
+## Overview
+
+The Operator SDK project recommends controller-runtime's [envtest][envtest] because it has a more active contributor community, it has become more mature than Operator SDK's test framework, and it does not require an actual cluster to run tests, which can be a huge benefit in CI scenarios. 
+
+## Using EnvTest
+
+You will see that `controllers/suite_test.go` is created. This file contains boilerplate for executing integration tests using [envtest][envtest] with [ginkgo](https://onsi.github.io/ginkgo/) and [gomega](https://onsi.github.io/gomega/).
+
+It means that you will implement your controller's tests in go using this stack and it will be called by the `suite_test.go` via go tool such as: 
+
+```shell
+go test ./test/my-e2e-test/ -v -ginkgo.v
+```
+
+To learn more about how you can test your controllers, see the documentation about [writing controller tests][writing-controller-tests].
+
+**NOTE** SDK project adopted this stack to perform its tests. In this way you might be able to check the tests implemented for the tool as well which can help you such as an example of its possibilities. See that [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) is covered by tests using the same stack as well. 
+
+## Other Options
```

Comment:
@dmesser I kind use your text here :-) 
Would be nice get your input on this one. The idea here is just we have the mainly/basic info which would be enough for now since it has been requested constantly. however, the community and us can improve it in the future.

Reply:
How about "Operator SDK adopted this stack to write tests for its operators. It might be useful to check upstream documentation and examples to learn how to better write tests for your operator. See [the controller runtime tests](link) for further examples."

Reply:
I tried to improve that based on your suggestion 👍. Tks

Reply:
this looks good